sdk/storage/storage-blob/src/models.ts

```diff
@@ -311,3 +311,19 @@ export interface HttpAuthorization {
    */
   value: string;
 }
+
+/** Known values of {@link BlobAudience} that the service accepts. */
+export const enum KnownBlobAudience {
```

Comment:
Our guidelines ban usage of `const enum` https://azure.github.io/azure-sdk/typescript_design.html#ts-no-const-enums

Please just use `enum`.

Reply:
Updated